### ISSUE TITLE
Extend the SELECT input field with an extra option "Other..." 

### ISSUE THREAD

**Issue by anna-geller** [kestra-io/kestra#4496]
### Feature description

It would be great to optionally allow the end user to provide an extra value, perhaps as a separate boolean property `other: true` (default should be set to `false`) so that when you click from the UI on the option "Other" it allows you to type a string:

```yaml
id: select
namespace: company.team

inputs:
  - id: pick_option
    type: SELECT
    defaults: one
    required: true
    other: true # <-- this would be new
    values:
      - one
      - two
      - three
tasks:
  - id: hello
    type: io.kestra.plugin.core.log.Log
    message: picked {{ inputs.pick_option }}
```

would render as:
![image](https://github.com/user-attachments/assets/9f5d6785-1a6e-43ef-b8dd-236e5bf1e239)

How Google Form does it:

![image](https://github.com/user-attachments/assets/c8299ba5-3f1e-4180-9341-317d5de1b03d)

**Comment by Skraye** [comment]
@anna-geller Should this also be added to the multiselect input ?

**Comment by anna-geller** [comment]
if the UI can handle it, that seems super useful!

**Comment by Skraye** [comment]
closed by #4884
